lib/stormpath-sdk/data_store.rb

```diff
@@ -141,7 +141,7 @@ def execute_request(http_method, href, resource=nil, query=nil)
 
       return if http_method == 'delete'
 
-      if result[HREF_PROP_NAME]
+      if result[HREF_PROP_NAME] and !resource.is_a? Stormpath::Provider::SamlMappingRules
```

Comment:
this is used for the /v1/attributeStatementMappingRules/$MAPPING_RULES_ID" request
check example request in the doc, just above the saml flow https://docs.stormpath.com/rest/product-guide/latest/auth_n#the-stormpath-saml-flow
problem is items don’t have the href property